packages/renderer/src/lib/preferences/PreferencesAuthenticationProvidersRendering.spec.ts

```diff
@@ -227,24 +240,28 @@ test('Expects images.icon option to be used when no themes are present', async (
 
   const icon = screen.getByRole('img', { name: `${testProvidersInfoWithSessionRequests[0].displayName}` });
   expect(icon).toBeInTheDocument();
-  expect(icon).toHaveAttribute('src', './icon.png');
+  expect(icon).toHaveAttribute('src', simpleIcon);
 });
 
 test('Expects images.icon.dark option to be used when theme is dark', async () => {
+  const darkIcon = './icon-dark.png';
   const providerWithImageIcon = [
     {
       id: 'test',
       displayName: 'Test Authentication Provider',
       accounts: [],
       images: {
         icon: {
-          dark: './icon-dark.png',
```

Comment:
you should extract this kind of refactoring to a separate very small PR that we can merge before this PR